manifests/Xunlei/XMP/6.1.7.810.yaml

```
@@ -0,0 +1,20 @@
+Id: XunLei.XMP
+Publisher: Xunlei Inc
+Name: XunLeiMediaPlayer
```

```suggestion
Id: XunLei.XunLeiMediaPlayer
Publisher: Xunlei
Name: XunLeiMediaPlayer
```
According to the manifest guideline, you could commit this suggestion and move this file into the `Xunlei/XunLeiMediaPlayer` directory in order to pass the validation test.